code/go/0chain.net/smartcontract/storagesc/config.go

```diff
@@ -382,7 +386,10 @@ func getConfiguredConfig() (conf *Config, err error) {
 		pfx + "writepool.max_lock_period")
 	// stake pool
 	conf.StakePool = new(stakePoolConfig)
-	conf.StakePool.MinLock = int64(scc.GetFloat64(pfx+"stakepool.min_lock") * 1e10)
+	conf.StakePool.MinLock, err = currency.ParseZCN(scc.GetFloat64(pfx + "stakepool.min_lock"))
```

Comment:
The MinLock here use ZCN unit, but zcnsc use SAS unit, we need to unify the unit, but can do it later.

Reply:
maybe we will have all these config changed to zcn, sas and be parsed